DEPLOY.md

```diff
@@ -10,14 +10,19 @@ This document will explain you how to run a local copy of Pan Docs.
   # Install mdbook using cargo
   cargo install mdbook
   # Remember to add cargo bin directory to your path
-  # Install python3 (on Debian systems)
+
+  # Install python3 (on Debian systems) make sure to install version 3.9
   apt install python3
+  # On windows version 3.9 can be installed from the Microsoft Store
   ```
 2. [Install the Python prerequisites](https://packaging.python.org/tutorials/installing-packages/#requirements-files).
 ```sh
 # Create and activate a virtualenv
 python -m venv env
+# bash
 source env/bin/activate
+# Windows
+. .\env\Scripts\Activate.bat # On powershell repalce bat with ps1
```

Comment:
I don't think there's any active maintainer on Windows, so I would strongly advice against adding windows-specific instructions nobody has the time nor the tools to keep maintained.

Reply:
Fair point, the Rust code checking for Windows got me think it might be a wanted change.
Since Windows is not supported I think it might be good to add that in the DEPLOY.md and simply remove this if in the Rust code.